assets/src/blocks/Carouselheader/CarouselHeaderBlock.js

```diff
@@ -150,7 +150,20 @@ export class CarouselHeaderBlock {
               link_url: '',
               link_url_new_tab: false,
             }
-          ]
+          ],
+          validation: slides => {
+            let isSlidesValid = true;
+            let errorMessages = [];
+
+            slides.forEach( (slide, index) => {
+              if (slide.image === null) {
+                isSlidesValid = false;
+                errorMessages.push(`Carousel Header Block: Slide ${ index + 1 } has no image`);
+              }
+            });
+      
+            return{isValid: isSlidesValid, messages: errorMessages}
```

Comment:
We really need to set up linting on this repo so that these are detected in the pipeline :stuck_out_tongue: Missing a space after `return` here.